public/app/plugins/datasource/jaeger/components/SearchForm.tsx

```diff
@@ -54,6 +54,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
       <InlineFieldRow>
         <InlineField label="Service" labelWidth={14} grow>
           <Select
+            aria-label="Service"
```

Comment:
`Select` accepts an `inputId` prop that `InlineField` checks so we don't have to repeat the label here